packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -508,12 +565,33 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
         super.dispose();
     }
 
+    /**
+     * Gets whether to use the colors option to colorize the line.
+     */
+    get useColors() {
+        return this._useColors;
+    }
+
     /**
      * Sets whether to use the colors option to colorize the line.
      * @param value true if use the colors, otherwise false
      */
-    public setUseColors(value: boolean) {
-        this._options.useColors = value;
+    set useColors(value: boolean) {
+        this._useColors = value;
+    }
+
+    /**
+     * Returns the colors used to colorize the line
+     */
+    get colors() {
+        return this._colors;
+    }
+
+    /**
+     * Sets the colors used to colorize the line
+     */
+    set colors(value: Nullable<Color3[]> | undefined) {

```

Comment:
Related to my previous comment - this `| undefined` is not needed.

Reply:
See my previous comment pls.

Reply:
Same here. value should not be "undefined". You can set it to be null, but there is no reason for someone to call "this.colos = undefined".

Reply:
this is a long discussion in a different thread on this page.
I understand the way it is, but it can be moved to be one or the other while staying flexible.